examples/vue3/cone-webxr.py

```diff
@@ -0,0 +1,111 @@
+from trame.app import get_server
+from trame.widgets import vuetify3, vtk as vtk_widgets
+from trame.ui.vuetify3 import SinglePageLayout
+
+from vtkmodules.vtkFiltersSources import vtkConeSource
+from vtkmodules.vtkRenderingCore import (
+    vtkRenderer,
+    vtkRenderWindow,
+    vtkRenderWindowInteractor,
+    vtkPolyDataMapper,
+    vtkActor,
+)
+
+# VTK factory initialization
+from vtkmodules.vtkInteractionStyle import vtkInteractorStyleSwitch  # noqa
+import vtkmodules.vtkRenderingOpenGL2  # noqa
+
+# -----------------------------------------------------------------------------
+# Trame initialization
+# -----------------------------------------------------------------------------
+
+server = get_server()
+server.client_type = "vue3"
+state, ctrl = server.state, server.controller
+
+state.trame__title = "VTK Local rendering"
+state.xr_active = False
+
+DEFAULT_RESOLUTION = 6
+
+# -----------------------------------------------------------------------------
+# VTK code
+# -----------------------------------------------------------------------------
+
+renderer = vtkRenderer()
+renderer.SetBackground(0.0231, 0.4194, 0.5592)
+renderWindow = vtkRenderWindow()
+renderWindow.AddRenderer(renderer)
+renderWindow.OffScreenRenderingOn()  # Prevent popup window
+
+renderWindowInteractor = vtkRenderWindowInteractor()
+renderWindowInteractor.SetRenderWindow(renderWindow)
+renderWindowInteractor.GetInteractorStyle().SetCurrentStyleToTrackballCamera()
+
+cone_source = vtkConeSource()
+mapper = vtkPolyDataMapper()
+actor = vtkActor()
+mapper.SetInputConnection(cone_source.GetOutputPort())
+actor.SetMapper(mapper)
+renderer.AddActor(actor)
+renderer.ResetCamera()
+renderWindow.Render()
+
+
+@state.change("resolution")
+def update_cone(resolution=DEFAULT_RESOLUTION, **kwargs):
+    cone_source.SetResolution(resolution)
+    ctrl.view_update()
+
+
+def update_reset_resolution():
+    state.resolution = DEFAULT_RESOLUTION
+
+
+def togglexr():
```

Comment:
`toggle_xr` or `toggleXR`

Reply:
Apologies for the confusion, I've been told (@luciemac ) that trame favors the `toggle_xr` notation. Please apply that convention in the other places of your code.